src/skin/legacyskinparser.cpp

```diff
@@ -1785,10 +1785,10 @@ void LegacySkinParser::setupSize(const QDomNode& node, QWidget* pWidget) {
             pWidget->setMinimumWidth(x);
         } else if (y >= 0) {
             pWidget->setMinimumHeight(y);
-        } else {
+        } else if (y != -1) {
```

Comment:
|| x != -1 ?

Reply:
Ah yes. Now -5, 5 will be accepted without a warnung. Ok?
Or should we go back to the version without any warning.
Is there a real use case for the warning?

Reply:
hm it definitely helps debugging as it points to the file and the faulty line.
btw: I'd love to have such a debugging message for the css parser